extensions/metrics-cluster-feature/src/metrics-settings.tsx

```diff
@@ -81,7 +81,7 @@ export class MetricsSettings extends React.Component<Props> {
 
   @computed get isTogglable() {
     if (this.inProgress) return false;
```

Comment:
Nit: maybe do one chain?

```suggestion
    return (
      !this.inProgress
      && !this.canUpgrade
      && this.isActiveMetricsProvider
      && this.props.cluster.status.phase === "connected"
    );
```